Duvet/Configuration/SheetConfiguration.swift

```diff
@@ -19,6 +19,10 @@ public struct SheetConfiguration: Equatable {
     /// The sheets initial position.
     public var initialPosition: SheetPosition
 
+    /// The color of the optional keyboard shim. If no value is provided the keyboard shim is not
```

Comment:
Not loving the use of the word "shim" for this. Makes me think of this: https://en.wikipedia.org/wiki/Shim_(computing)

Reply:
Oof! Good catch, I should change that to "keyboard background".